integ-tests/src/cucumber/resources/features/dataservice/create_environment_target_revision.feature

```diff
@@ -0,0 +1,21 @@
+@ignore
+@dataservice
+@create-environment-target-revision
+Feature: Create environment target revision
+
+  Scenario: Create an environment target revision
+    Given I create an environment
+    When I create an environment target revision
+    Then the created environment target revision response is valid
+
+  Scenario: Create an environment target revision for a non-existent environment
+    When I create an environment target revision
+    Then there should be an EnvironmentNotFoundException thrown
+
+  Scenario: Create an environment target revision that already exists
+    Given I create an environment
+    And I create an environment target revision
+    When I try to create another environment target revision with the same id and version
+    Then there should be an EnvironmentTargetRevisionExistsException thrown
```

Comment:
This specification looks correct to me, and is what I would expect from an API call that creates a new revision of an environment. However, as far as I can tell from the code, the `EnvironmentTargetRevision` resource in the data service created by `createEnvironmentTargetRevision` is the one that *can not* have duplicate records for the same environment name, even if the `environmentVersion` is different.